pyphare/pyphare/pharein/restarts.py

```diff
@@ -9,20 +9,41 @@
 
 def validate(sim):
 
-    sim.restart_options["timestamps"] = phare_utilities.np_array_ify(sim.restart_options["timestamps"])
-    init_time = sim.start_time()
-
-    timestamps = sim.restart_options["timestamps"]
-    if np.any(timestamps < init_time):
-        raise RuntimeError(f"Error: timestamp({sim.time_step_nbr}) cannot be less than simulation.init_time({init_time}))")
-    if np.any(timestamps > sim.final_time):
-        raise RuntimeError(f"Error: timestamp({sim.time_step_nbr}) cannot be greater than simulation.final_time({sim.final_time}))")
-    if not np.all(np.diff(timestamps) >= 0):
-        raise RuntimeError(f"Error: {clazz}.{key} not in ascending order)")
-    if not np.all(np.abs(timestamps / sim.time_step - np.rint(timestamps/sim.time_step) < 1e-9)):
-        raise RuntimeError(f"Error: {clazz}.{key} is inconsistent with simulation.time_step")
-
-    sim.restart_options["timestamps"] = conserve_existing(sim)
+    restart_options = sim.restart_options
+
+    if "elapsed_timestamps" in restart_options:
+        import datetime
+        restart_options["elapsed_timestamps"] = [
+            int(ts.total_seconds()) if isinstance(ts, datetime.timedelta) else ts
+            for ts in phare_utilities.np_array_ify(restart_options["elapsed_timestamps"])]
+
+        if not np.all(np.diff(restart_options["elapsed_timestamps"]) >= 0):
```

Comment:
I don't think we should accept 0 seconds as a valid interval, do you?

Reply:
if you only configure elapsed timestamps. it's essentially asking for the init dump, not sure it's bad